### PR TITLE
Minor Updations to your README.md content

### PR DESCRIPTION
**Pull request by Git-Harshit**
Minor changes to your README file. Please have a look, and merge if it seems fine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,8 @@ Future Projects :<br>
   - _Opamp design and analysis, Two and multiple stages in sky130 + ptm models(Lower nodes) + cadence virtuoso_
   - _Voltage reference circuit with sky130pdk_
   - _More to come...._
+
+
+For anything further, feel free to reach out! 🙂
+
+### Visitor Count: ![Visitor Count](https://profile-counter.glitch.me/D-curs-D/count.svg)
```
Comment (D-curs-D) [inline README.md]: Visitor Count not necessary, since it's not accurate. 
Can you remove this?

File: README.md [hunk 2]
```diff
@@ -25,3 +25,8 @@ Future Projects :<br>
   - _Opamp design and analysis, Two and multiple stages in sky130 + ptm models(Lower nodes) + cadence virtuoso_
   - _Voltage reference circuit with sky130pdk_
   - _More to come...._
+
+
+For anything further, feel free to reach out! 🙂
```
Comment (D-curs-D) [inline README.md]: Reach out seems fine, but I don't think I want it rn.

**Review by D-curs-D — COMMENTED** [review]
Thank you, I have proposed a couple changes please go through them and revert back

File: README.md (hunk 1 above)
Comment (Git-Harshit) [reply README.md]: Sure

**Review by D-curs-D — APPROVED** [review]

**Pull request merged**